src/cms-content/region-overrides/index.ts

```diff
@@ -0,0 +1,67 @@
+import RegionOverrides from './region-overrides.json';
+import { Markdown } from 'cms-content/utils';
+import { getMetricDefinition } from 'common/metric';
+import { Metric } from 'common/metricEnum';
+import regions, { Region } from 'common/regions';
+
+type Include = 'region' | 'subregions' | 'region-and-subregions';
+
+interface RegionOverride {
+  include: Include;
+  metric: string;
+  region: string;
+  context: string;
+  blocked: boolean;
+  disclaimer?: Markdown;
+}
+
+/** Looks for any override applied to the specified region + metric and returns it. */
+export function getRegionMetricOverride(
+  region: Region,
+  metric: Metric,
+): RegionOverride | undefined {
+  const metricDefinition = getMetricDefinition(metric);
+  for (const override of getOverridesForRegion(region)) {
+    if (override.metric === metricDefinition.metricId) {
+      return override;
+    }
+  }
+  return undefined;
+}
```

Comment:
Consider creating a map between CMS metric IDs and the values in the enum instead - I feel like this might make it easier to end up with a circular dependency (which is close to happen in metrics, metric definitions, etc).

```ts
const metricByOverrideMetricId = {
  'metric.caseDensity': Metric.CASE_DENSITY,
  // ...
}

function getRegionMetricOverride(region: Region, metric: Metric) {

}
```

I think this is better because each individual metric will need to call this file to render elements, for example, in `common/metrics/case_density.tsx`:

```tsx
// part of the metric definition...
function renderDisclaimer(region: Region) {
  if (getRegionMetricOverride(Metric.CASE_DENSITY)) {
    return "...";
  } else {
    return "...";
  }
}
```

Reply:
Thanks.  In my mind it seemed useful to give metrics a persistent string identifier that we could use, but we don't actually need it and this ends up cleaner.

Reply:
oh yeah, I think having a string for each metric might be useful, I'm a bit concerned about the dependencies between files